pkg/storaged/client.js

```diff
@@ -25,6 +25,9 @@
 
     var utils = require('./utils');
 
+    var nfs_mounts_py = require("raw!./nfs-mounts.py");
+    var pyinvoke = [ "sh", "-ec", "exec $(which python3 2>/dev/null || which python) $@", "--", "-" ];
```

Comment:
One of these days this should really move to lib/, given that we copy this around more and more :-)

Reply:
Yes.

In this case, the 'golden' solution would be to replace the whole Python script with new API in udisks2, I'd say.  I want to work on that as well.  We might decide that this PR here is only a proof-of-concept that we are not going to merge.  What do you think?

Reply:
Done, #7852